tests/integration/preprocessor/_regrid/test_regrid.py

```diff
@@ -35,9 +35,69 @@ def test_regrid__linear(self):
                                     coord_system=self.cs)
         coords_spec = [(lats, 0), (lons, 1)]
         grid = iris.cube.Cube(data, dim_coords_and_dims=coords_spec)
-        result = regrid(self.cube, grid, 'linear')
+        self.grid_for_linear = grid
+
+        # Setup unstructed cube and grid
+        data = np.empty((1, 1))
```

Comment:
```suggestion
        data = np.zeros((1, 1))
```
for reproducability.

Reply:
That was copy-pasted from existing code, and I really do not want to mess with other tests

Reply:
It shows up as your code, hence you are responsible.